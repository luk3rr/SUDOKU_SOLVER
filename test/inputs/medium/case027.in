000000290 008000045 720040600 800301007 000000000 570006000 601490003 007605002 040000006 
