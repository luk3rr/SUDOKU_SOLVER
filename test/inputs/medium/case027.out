Solving the following grid:
0 0 0 | 0 0 0 | 2 9 0 
0 0 8 | 0 0 0 | 0 4 5 
7 2 0 | 0 4 0 | 6 0 0 
------+-------+------
8 0 0 | 3 0 1 | 0 0 7 
0 0 0 | 0 0 0 | 0 0 0 
5 7 0 | 0 0 6 | 0 0 0 
------+-------+------
6 0 1 | 4 9 0 | 0 0 3 
0 0 7 | 6 0 5 | 0 0 2 
0 4 0 | 0 0 0 | 0 0 6 


Solution found :')

4 6 5 | 1 7 3 | 2 9 8 
1 3 8 | 2 6 9 | 7 4 5 
7 2 9 | 5 4 8 | 6 3 1 
------+-------+------
8 9 4 | 3 2 1 | 5 6 7 
2 1 6 | 7 5 4 | 3 8 9 
5 7 3 | 9 8 6 | 1 2 4 
------+-------+------
6 5 1 | 4 9 2 | 8 7 3 
9 8 7 | 6 3 5 | 4 1 2 
3 4 2 | 8 1 7 | 9 5 6 

Algorithm: BFS
Total time: 290 ms
Total expanded states: 42557
